app/views/authors/show.html.haml

```diff
@@ -0,0 +1,9 @@
+.author
+  .name
+    = @author.full_name
+  .biography= @author.biography
+  %ul.author-books
+    - @author.last_5_books.each do |book|
+      %li.book
+        = link_to book_path(book) do
+          = image_tag book.cover_url
```

Comment:
@Fatpaher Read task carefully. Books should be displayed as on list page: https://cdn.discourse.org/business/uploads/upcase/original/2X/b/b42fc7181a749025b338e21780411ca7e3a8c60b.jpg